cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -144,6 +150,16 @@ def _onchange_variable_id(self):
             else:
                 rec.option_id = None
 
+    @api.depends("variable_id")
```

Comment:
What if variable access level has changed? Eg it was "2" and then became "1". This means "User" group members will not be able to access the variable values although they should.